tools/crm_resource.c

```diff
@@ -29,12 +29,46 @@
 
 #define SUMMARY "crm_resource - perform tasks related to Pacemaker cluster resources"
 
+enum rsc_command {
+    cmd_none = 0,           // No command option given (yet)
+    cmd_ban,
+    cmd_cleanup,
+    cmd_clear,
+    cmd_colocations,
+    cmd_colocations_deep,
+    cmd_cts,
+    cmd_delete,
+    cmd_delete_param,
+    cmd_execute_agent,
+    cmd_fail,
+    cmd_get_param,
+    cmd_get_property,
+    cmd_list_active_ops,
+    cmd_list_agents,
+    cmd_list_all_ops,
+    cmd_list_alternatives,
+    cmd_list_instances,
+    cmd_list_providers,
+    cmd_list_resources,
+    cmd_list_standards,
+    cmd_locate,
+    cmd_metadata,
+    cmd_move,
+    cmd_query_raw_xml,
+    cmd_query_xml,
+    cmd_refresh,
+    cmd_restart,
+    cmd_set_param,
+    cmd_set_property,
+    cmd_wait,
+    cmd_why,
```

Comment:
This is going to make it so much easier to follow what's going on for all the crm_resource changes I've been working on.